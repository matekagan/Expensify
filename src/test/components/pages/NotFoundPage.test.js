import React from 'react';
import { shallow } from 'enzyme';
import NotFoundPage from '../../../main/components/pages/NotFoundPage';

test('Should render 404 not found page', () => {
    const wrapper = shallow(<NotFoundPage />);
    expect(wrapper).toMatchSnapshot();
});
