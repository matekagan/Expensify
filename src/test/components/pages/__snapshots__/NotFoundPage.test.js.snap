// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Should render 404 not found page 1`] = `
<div>
  404 -
  <Link
    replace={false}
    to="/"
  >
    Home page
  </Link>
</div>
`;
